src/PlcNodeManager.cs

```diff
@@ -453,7 +455,7 @@ private FolderState AddMethods(FolderState methodsFolder)
             return methodsFolder;
         }
 
-        private void UpdateNodes(BaseDataVariableState[] nodes, NodeType type, StatusCode status, bool addBadValue)
+        public void UpdateNodes(BaseDataVariableState[] nodes, NodeType type, StatusCode status, bool addBadValue)

```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

wouldn't be a bit cleaner to have a separate NodeManager for the Guid nodes like we have to the Alarms, Events, ReferenceServer, etc? That way wou could keep the PlcNodeManager untouched.

Reply:
The "guid nodes" is a very small feature. My idea is that we use the PlcNodeManager for all the small features. We can discuss after vacation with Hans and decide on a simple and nice architecture, and then refactor. I will start encapsulating the small features in preparation for the refactoring.

Reply:
sounds good! thanks for the clarification Luis